README.md

```diff
@@ -37,42 +37,14 @@ In fact, this pattern is used for various types, namely:
 
 (Naturally, when the API specification is actually modelling types to be unions (e.g. `InlineQueryResult`), this is reflected here as a union type, too.)
 
-## Available Methods
-
-In addition to the types, this package provides you with another type `Telegram` which contains all available **methods** of the API.
-There is no further structure applied to this.
-
-Each method takes just a single argument with a structure that corresponds to the object expected by Telegram.
-If you need to directly access that type, consider using `Opts<M>` where `M` is the method name (e.g. `Opts<'getMe'>`).
-
-Note that `Opts<M>` will give you an empty object type (i.e. `{}`) for methods that do not take any parameters.
-That is to say, it will not give you a type error or `undefined` (as opposed to something like `Parameters<Telegram['getMe']>[0]`).
-
-Each method returns the object that is specified by Telegram.
-If you directly need to access the return type of a method, consider using `Ret<M>` where `M` is the method name (e.g. `Opts<'getMe'>`).
```

Comment:
Why did you remove this section? The methods are still there, no?

Reply:
I removed it because `Telegram` no longer exists; `Opts` needs `<F>`. This is confusing before you read about "Customizing `InputFile` and Accessing API Methods" which covers this.

Reply:
Alright.